File: Arrays and Lists/exmp.js
// Build an array of 5 patient names
let patients = ["Ahmed", "Abdullah", "General", "Eminent", "David"];

// Add one patient to the end
patients.push("Aisha");

// Insert one patient
patients.unshift("Khadijah");

// Remove one patient
patients.splice(patients.indexOf("General"), 1);

// Print the final count
console.log("Final count:", patients.length);

// Print the last item
console.log("Last item:", patients[patients.length - 1]);

File: Arrays and Lists/exmp.py
# Build a list of 5 patient names
patients = ["Ahmed", "Abdullah", "General", "Eminent", "Mahbub"]

# Add one patient to the end
patients.append("Aisha")

# Insert one patient
patients.insert(2, "Khadijah")

# Remove one patient
patients.remove("General")

# Print the final count
print("Final count:", len(patients))

# Print the last item
print("Last item:", patients[-1])